core-api/src/services/access.service.ts

```diff
@@ -0,0 +1,41 @@
+import db from "../config/database";
+import { Access, SafeAccess } from "../entities/access.entity";
+import { ServerError, ErrorCode } from "../utils/serverError";
+import { ValidationError, validate } from "class-validator";
+import Messages from "../utils/messages";
+
+/**
+ * @namespace Services
+ * @class AccessService
+ */
+export class AccessService {
+    /**
+     * Creates a new Access in the database
+     * @param place
+     * @param permission
+     */
+    public static async create(place: Place, permission: Permission): Promise<SafeAccess> {
```

Comment:
This function probably will receive only number or string when we actually use this API, so try to exchange this parameters for place_id and permissions_id

Reply:
See the way implemented on placeCategory

Reply:
Changes done

Reply:
Changes done